### PR TITLE
Add county tests data

### PR DESCRIPTION
**Pull request by igorkofman**
This PR addresses issue #<insert number>

### Please Check
- [ ] Are the [JSON schemas](https://github.com/covid-projections/covid-data-model/tree/master/api/schemas) updated?
- [ ] Are the [api docs](https://github.com/covid-projections/covid-data-model/blob/master/api/README.V1.md) updated?
- [ ] Are tests passing?

### CODE REVIEW

File: libs/datasets/sources/cds_dataset.py [hunk 1]
```diff
@@ -112,8 +123,17 @@ def standardize_data(cls, data: pd.DataFrame) -> pd.DataFrame:
...
+        data = data[data['country'] == 'United States']
+        data['state_abbr'] = data[cls.Fields.STATE].apply(lambda x: US_STATE_ABBREV[x] if x in US_STATE_ABBREV else x)
+        data['state_tmp'] = data['state']
+        data['state'] = data['state_abbr']
+
         fips_data = dataset_utils.build_fips_data_frame()
         data = dataset_utils.add_fips_using_county(data, fips_data)
 
+        # put the state column back
+        data['state'] = data['state_tmp']
```
Comment (fridiculous) [inline libs/datasets/sources/cds_dataset.py:137]: wait, why are we moving it back and forth?

File: libs/functions/generate_api.py [hunk 2]
```diff
@@ -200,10 +207,18 @@ def generate_state_timeseries(
...
     state_abbrev = US_STATE_ABBREV[projection_row[rc.STATE_FULL_NAME]]
     fips = projection_row[rc.FIPS]
-    can_dataseries = get_can_projection.get_can_raw_data(
+
+    raw_dataseries = get_can_projection.get_can_raw_data(
         input_dir, state_abbrev, fips, AggregationLevel.COUNTY, intervention
     )
 
+    testing_df = get_testing_timeseries_by_fips(fips)
+    new_df = pd.DataFrame(raw_dataseries).merge(
```
Comment (fridiculous) [inline libs/functions/generate_api.py:216]: 👍

File: libs/functions/generate_api.py [hunk 3]
```diff
@@ -143,6 +146,10 @@ def _generate_state_timeseries_row(json_data_row):
 
 
 def _generate_county_timeseries_row(json_data_row):
+    tested = _get_or_none(json_data_row[CDSDataset.Fields.TESTED])
+    cases = _get_or_none(json_data_row[CDSDataset.Fields.CASES])
+    negative = tested and cases and (tested - cases)
```
Comment (fridiculous) [inline libs/functions/generate_api.py:151]: sorry, what's the logic?

Comment (igorkofman) [reply libs/functions/generate_api.py:151]: negative is null unless we have tested and cases, in that case it's tested - cases

**Pull request merged**